plugins/out_http/http_conf.c

```diff
@@ -81,6 +85,36 @@ struct flb_out_http *flb_http_conf_create(struct flb_output_instance *ins,
         flb_output_net_default("127.0.0.1", 80, ins);
     }
 
+    /* Check if AWS SigV4 authentication is enabled */
+#ifdef FLB_HAVE_SIGNV4
+#ifdef FLB_HAVE_AWS
+    if (ctx->has_aws_auth) {
+        ctx->aws_service = flb_output_get_property("aws_service", ctx->ins);
```

Comment:
Why we cannot set aws_service in  FLB_AWS_CREDENTIAL_BASE_CONFIG_MAP as well and put it in lb_managed_chain_provider_create

Reply:
the managed_chain_provider_create is meant to be a general credential provider base init for all plugins using credential services. We can easily migrate cloudwatch, firehose, kinesis, s3, and es, to use this base as well since they all share the same 4 output properties.

aws_service is only used for sigv4, and doesn't generalize well to be part of the credential provider.